include/l3.h

```diff
@@ -47,6 +47,18 @@
 extern "C" {
 #endif
 
+/**
+ * Different logging interfaces, mainly intended for use by micro-benchmarking
+ * utility programs.
+ */
+typedef enum {
+      L3_LOG_UNDEF      = 0
+    , L3_LOG_MMAP
+    , L3_LOG_FPRINTF
```

Comment:
Adding this to lay down the ground-work to extend existing client-server performance tests to run with diff logging schemes.

We may, well, choose to take this out later, but for now, this change seems useful to enable further dev of perf u-benchmarking programs.